<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-J08G42B1" failures="5" tests="8" name="Test Two" time="0.099" errors="0" timestamp="2025-03-23T15:12:57 IST">
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test" time="0.009"/>
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test1" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [VIT] but found [MIT]">
      <![CDATA[java.lang.AssertionError: expected [VIT] but found [MIT]
at com.testngsamples.Samples.Example8_2.test1(Example8_2.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test2" time="0.002">
    <failure type="java.lang.AssertionError" message="Test to check title expected [VIT] but found [MIT]">
      <![CDATA[java.lang.AssertionError: Test to check title expected [VIT] but found [MIT]
at com.testngsamples.Samples.Example8_2.test2(Example8_2.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test3" time="0.001"/>
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test4" time="0.003">
    <failure type="java.lang.AssertionError" message="Test to check login credentials expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Test to check login credentials expected [false] but found [true]
at com.testngsamples.Samples.Example8_2.test4(Example8_2.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test5" time="0.002"/>
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test6" time="0.001">
    <failure type="java.lang.AssertionError" message="Test to check login credentials expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test to check login credentials expected [true] but found [false]
at com.testngsamples.Samples.Example8_2.test6(Example8_2.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
  <testcase classname="com.testngsamples.Samples.Example8_2" name="test8" time="0.002">
    <failure type="java.lang.AssertionError" message="Test to check login credentials expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Test to check login credentials expected [false] but found [true]
at com.testngsamples.Samples.Example8_2.test8(Example8_2.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- test8 -->
</testsuite> <!-- Test Two -->
